sources/packages/backend/libs/test-utils/src/factories/offering-change-request.ts

```diff
@@ -0,0 +1,31 @@
+import { EducationProgramOffering, OfferingStatus } from "@sims/sims-db";
+
+/**
+ * Create a fake offering request change.
+ * @param relations dependencies:
+ *  - `currentOffering` current offering, which is requested for the changed.
+ * @returns the current and requested offering.
+ */
+export function createFakeOfferingRequestChange(relations: {
+  currentOffering: EducationProgramOffering;
```

Comment:
I believe that this would be a better fit on the `options` than on the `relations` because it is a parameter used as a baseline to set, for instance, `parentOffering`, and `precedingOffering`.